src/demo/demo.stache

```diff
@@ -25,7 +25,7 @@
 			<div class="block slide {{#is ../activeSlideIndex scope.index}} active {{/is}}" tabindex="{{#is ../activeSlideIndex scope.index}} 0 {{else}} -1 {{/is}}" style="background-color: {{./backgroundColor}}">
 				
 				<div class="slideContent">
-					<h1>{{./title}}</h1>
+					<h1>{{{./title}}}</h1>
```

Comment:
Isn't this a bit unsafe?

Reply:
Not too sure. What I do know is that the content that will be populating these tags is coming from our CMS backend so at least it is trusted. Also, it's the only way I know of allowing html encoded in our backend to display not as a string but as html in our template.